Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -552,7 +552,11 @@ def merge_bottle_spec(old_bottle_spec, new_bottle_hash)
       old_value = old_bottle_spec.send(key)
       next if key == :rebuild && old_value.zero?
       next if key == :prefix && old_value == Homebrew::DEFAULT_PREFIX
-      next if key == :cellar && old_value == Homebrew::DEFAULT_REPOSITORY
```

Comment:
Would be good to have the same thing for these too: https://github.com/Homebrew/brew/blob/e69a54d0d87889b691ba9d201cb768043af90495/Library/Homebrew/global.rb#L64-L69

Reply:
Should we use these instead? https://github.com/Homebrew/brew/blob/e69a54d0d87889b691ba9d201cb768043af90495/Library/Homebrew/global.rb#L87-L89

Reply:
We should check them for every platform so we never set it even if we're running `brew bottle` on a different platform to the platform it's bottling for (e.g. when merging).

Reply:
Ah, right, I didn't think of that